### ISSUE TITLE
New specs to support retpo kernels

### ISSUE THREAD

**Issue by xiangce** [RedHatInsights/insights-frontend-assets#12]
From https://github.com/RedHatInsights/insights-core/pull/1057

~~~
cpu_vulns_meltdown = simple_file("sys/devices/system/cpu/vulnerabilities/meltdown")
cpu_vulns_spectre_v1 = simple_file("sys/devices/system/cpu/vulnerabilities/spectre_v1")
cpu_vulns_spectre_v2 = simple_file("sys/devices/system/cpu/vulnerabilities/spectre_v2")
x86_retp_enabled = simple_file("sys/kernel/debug/x86/retp_enabled")
~~~

**Comment by kylape** [comment]
Resolved in 121e151

**Comment by xiangce** [comment]
Thanks!
